packages/elastic-datemath/BUILD.bazel

```diff
@@ -39,14 +43,15 @@ ts_config(
 )
 
 ts_project(
-  name = "tsc",
+  name = "tsc_types",
   args = ['--pretty'],
   srcs = SRCS,
   deps = DEPS,
   declaration = True,
   declaration_map = True,
-  incremental = True,
-  out_dir = "target",
+  emit_declaration_only = True,
+  incremental = False,
+  out_dir = "target_types",
   source_map = True,
```

Comment:
Are source maps still needed?

Reply:
At least for now they are still needed to map the "point and click" behaviour on the IDEs. Once we remove the ts_project rules they are not needed anymore